clients/src/main/java/org/apache/kafka/common/config/AbstractConfig.java

```diff
@@ -360,4 +360,27 @@ public V get(Object key) {
             return super.get(key);
         }
     }
+
+    /**
+     * Postprocess the configuration so that exponential backoff is disabled if the reconnect backoff
+     * is configured, but the maximum reconnect backoff is not configured.
+     *
+     * @param parsedValues          The parsedValues as provided to postProcessParsedConfig.
+     * @param reconnectBackoff      The name of the configuraiton key for reconnect backoff.
+     * @param reconnectBackoffMax   The name of the configuraiton key for maximum reconnect backoff.
+     * @return                      The new values which have been set as described in postProcessParsedConfig.
+     */
+    protected Map<String, Object> postProcessReconnectBackoffConfigs(Map<String, Object> parsedValues,
```

Comment:
`originals()` is public:

```java
public Map<String, Object> originals() {
```

It's not generic if `reconnectBackoff` and `reconnectBackoffMax` are mentioned. :) Also, `AbstractConfig` is meant to be subclassed, so `protected` methods are public API and cannot be changed.